templates/galaxy-handler@.service.j2

```diff
@@ -1,5 +1,5 @@
 [Unit]
-Description=Galaxy Handlers
+Description=Galaxy Handlers for {{galaxy_systemd_sub_dest}}
```

Comment:
This is probably ok, we often call it an `galaxy_instance_codename` internally. Maybe something like
```suggestion
Description=Galaxy Handlers{% if galaxy_instance_codename %} for {{ galaxy_instance_codename | default(galaxy_systemd_unit_name) }}{% endif %}
```

Reply:
This will be cleaner yes